Bottom: 765c0028f2a0bb6764bcdcb05c3a4a0e39176aea
Top:    e6acb136efe2b3005c6d439a3334ccb5045f70fb
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:15:37 +0000

Refresh of remove-left-over-debug

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index ba5c857..3d525eb 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1503,8 +1503,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                               copper_appearance,
                               NULL));
-
-    //break;
   }
